JavaScript/JavaScriptSDK.Tests/CheckinTests/Sender.tests.ts

```diff
@@ -38,12 +38,19 @@ class SenderTests extends TestClass {
             emitLineDelimitedJson: () => this.emitLineDelimitedJson,
             maxBatchSizeInBytes: () => this.maxBatchSizeInBytes,
             maxBatchInterval: () => this.maxBatchInterval,
-            disableTelemetry: () => this.disableTelemetry
+            disableTelemetry: () => this.disableTelemetry,
+            storeSendBufferInSessionStorage: () => false
         };
 
-        this.getSender = () => new Microsoft.ApplicationInsights.Sender(config);
-        this.errorSpy = this.sandbox.spy(Microsoft.ApplicationInsights.Sender, "_onError");
-        this.successSpy = this.sandbox.spy(Microsoft.ApplicationInsights.Sender, "_onSuccess");
+        this.getSender = () => {
+            var sender = new Microsoft.ApplicationInsights.Sender(config);
+
+            this.errorSpy = this.sandbox.spy(sender, "_onError");
+            this.successSpy = this.sandbox.spy(sender, "_onSuccess");
```

Comment:
P.S. If the old way was making global handlers that ran for any Sender instance then that seems safer than redefining the properties on each call for a specific instance.  What happens when two calls come in to getSender and are actively used in parallel?